src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const interArr = sourceString.split(';');
```

Comment:
[CODE STYLE] - Avoid variable namings like `array`, `object` `arr`, `obj`, `result`. It will decrease code readability. Consider using a name that reflects the content of the array or the purpose of the variable, such as `styleDeclarations` or `styleEntries`.